Auto update failing for vulkan-tools

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the vulkan-tools package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating vulkan-tools [Current version: 1.3.250]
INFO: package being updated to 1.3.251.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  768k    0  768k    0     0   790k      0 --:--:-- --:--:-- --:--:--  790k

You are about to commit these changes:

--------------------
diff --git a/packages/vulkan-tools/build.sh b/packages/vulkan-tools/build.sh
index ccc1fffb6..60c57ca23 100644
--- a/packages/vulkan-tools/build.sh
+++ b/packages/vulkan-tools/build.sh
@@ -3,9 +3,9 @@ TERMUX_PKG_DESCRIPTION="Vulkan Tools and Utilities"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # This package and vulkan-headers should be updated at same time. Otherwise, they do not compile successfully.
-TERMUX_PKG_VERSION="1.3.250"
+TERMUX_PKG_VERSION="1.3.251"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-Tools/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=a1cd4f7d8d58bec26efb1e5ea73f82dcf36a9662d55401b81a783071b211a35a
+TERMUX_PKG_SHA256=03851334d9751464f96b2d3c49cac884b7be5178c04d162a445bb2ca93f56519
 TERMUX_PKG_BUILD_DEPENDS="libwayland-protocols, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-loader-generic (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_DEPENDS="libc++, libx11, libxcb, libwayland"
 TERMUX_PKG_RECOMMENDS="vulkan-loader"
--------------------

bump(main/vulkan-tools): 1.3.251


Not committing to Git!

INFO: Trying to build package.
INFO: Package vulkan-tools exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building vulkan-tools for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  45882      0 --:--:-- --:--:-- --:--:-- 45923
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1358      0 --:--:-- --:--:-- --:--:--  1356
100   833  100   833    0     0   1358      0 --:--:-- --:--:-- --:--:--  1356
gpg: Signature made Wed 31 May 2023 06:22:37 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  21214      0 --:--:-- --:--:-- --:--:-- 21244
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1389      0 --:--:-- --:--:-- --:--:--  1390
gpg: Signature made Sun 14 May 2023 09:34:01 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  21263      0 --:--:-- --:--:-- --:--:-- 21264
100 13120  100 13120    0     0  21260      0 --:--:-- --:--:-- --:--:-- 21229
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1328      0 --:--:-- --:--:-- --:--:--  1328
gpg: Signature made Sun 28 May 2023 06:46:55 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@25c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libc++/libc++_25c_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 18  216k   18 41673    0     0  54578      0  0:00:04 --:--:--  0:00:04 54545
100  216k  100  216k    0     0   232k      0 --:--:-- --:--:-- --:--:--  232k
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@28-3 if necessary...
Skipping already built dependency libandroid-support@28-3
Downloading dependency libexpat@2.5.0 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libe/libexpat/libexpat_2.5.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 81732  100 81732    0     0   614k      0 --:--:-- --:--:-- --:--:--  618k
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.4-1 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libf/libffi/libffi_3.4.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30868  100 30868    0     0   271k      0 --:--:-- --:--:-- --:--:--  274k
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libiconv/libiconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  543k  100  543k    0     0  3698k      0 --:--:-- --:--:-- --:--:-- 3721k
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/i/iconv/iconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8988  100  8988    0     0  14907      0 --:--:-- --:--:-- --:--:-- 14930
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.4.3 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblzma/liblzma_5.4.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  238k  100  238k    0     0   385k      0 --:--:-- --:--:-- --:--:--  385k
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.4.3 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xz-utils/xz-utils_5.4.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 63452  100 63452    0     0   405k      0 --:--:-- --:--:-- --:--:--  407k
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.4-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses/ncurses_6.4-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  502k  100  502k    0     0   783k      0 --:--:-- --:--:-- --:--:--  783k
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.4-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-utils/ncurses-utils_6.4-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 86024  100 86024    0     0   675k      0 --:--:-- --:--:-- --:--:--  677k
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.4-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs/ncurses-ui-libs_6.4-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 32276  100 32276    0     0   281k      0 --:--:-- --:--:-- --:--:--  283k
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.4-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs-static/ncurses-ui-libs-static_6.4-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39044  100 39044    0     0  42778      0 --:--:-- --:--:-- --:--:-- 42811
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.1 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/readline/readline_8.2.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  227k  100  227k    0     0  1663k      0 --:--:-- --:--:-- --:--:-- 1676k
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.23 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gdbm/gdbm_1.23_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  138k  100  138k    0     0  1025k      0 --:--:-- --:--:-- --:--:-- 1026k
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-posix-semaphore/libandroid-posix-semaphore_0.1-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  4128  100  4128    0     0  28543      0 --:--:-- --:--:-- --:--:-- 28666
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libb/libbz2/libbz2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 26634  100 26634    0     0  34927      0 --:--:-- --:--:-- --:--:-- 34906
100 26634  100 26634    0     0  34923      0 --:--:-- --:--:-- --:--:-- 34906
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/bzip2/bzip2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 56 25506   56 14531    0     0  18695      0  0:00:01 --:--:--  0:00:01 18677
100 25506  100 25506    0     0  32792      0 --:--:-- --:--:-- --:--:-- 32784
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2023.01.10 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates/ca-certificates_1:2023.01.10_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  115k  100  115k    0     0   857k      0 --:--:-- --:--:-- --:--:--  858k
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2023.01.10 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates-java/ca-certificates-java_1:2023.01.10_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  105k  100  105k    0     0   811k      0 --:--:-- --:--:-- --:--:--  815k
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.2.13 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/z/zlib/zlib_1.2.13_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 60060  100 60060    0     0   473k      0 --:--:-- --:--:-- --:--:--  476k
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.1.0 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl/openssl_1:3.1.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1557k  100 1557k    0     0  9781k      0 --:--:-- --:--:-- --:--:-- 9794k
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.1.0 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl-tool/openssl-tool_1:3.1.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  4  236k    4 10430    0     0  11525      0  0:00:21 --:--:--  0:00:21 11524
100  236k  100  236k    0     0   193k      0  0:00:01  0:00:01 --:--:--  193k
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libcrypt/libcrypt_0.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8720  100  8720    0     0  77997      0 --:--:-- --:--:-- --:--:-- 77857
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.42.0 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libsqlite/libsqlite_3.42.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  568k  100  568k    0     0  3870k      0 --:--:-- --:--:-- --:--:-- 3892k
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.42.0 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/sqlite/sqlite_3.42.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  2  534k    2 13170    0     0  14078      0  0:00:38 --:--:--  0:00:38 14070
100  534k  100  534k    0     0   374k      0  0:00:01  0:00:01 --:--:--  375k
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.13 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/tcl/tcl_8.6.13_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2191k  100 2191k    0     0  11.9M      0 --:--:-- --:--:-- --:--:-- 11.9M
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.0.9-1 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/brotli/brotli_1.0.9-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  296k  100  296k    0     0  2053k      0 --:--:-- --:--:-- --:--:-- 2060k
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.39 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libp/libpng/libpng_1.6.39_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  191k  100  191k    0     0  1395k      0 --:--:-- --:--:-- --:--:-- 1394k
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.0-1 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/freetype/freetype_2.13.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  397k  100  397k    0     0  2766k      0 --:--:-- --:--:-- --:--:-- 2777k
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/ttf-dejavu/ttf-dejavu_2.37-8_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2410k  100 2410k    0     0  13.8M      0 --:--:-- --:--:-- --:--:-- 13.9M
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.14.2-1 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fontconfig/fontconfig_2.14.2-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  115k  100  115k    0     0   890k      0 --:--:-- --:--:-- --:--:--  894k
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.14.2-1 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fontconfig-utils/fontconfig-utils_2.14.2-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 14336  100 14336    0     0  18517      0 --:--:-- --:--:-- --:--:-- 18498
100 14336  100 14336    0     0  18514      0 --:--:-- --:--:-- --:--:-- 18498
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.0 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xorg-util-macros/xorg-util-macros_1.20.0_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 22000  100 22000    0     0  28164      0 --:--:-- --:--:-- --:--:-- 28169
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2022.2 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xorgproto/xorgproto_2022.2_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 16  240k   16 41620    0     0  38500      0  0:00:06  0:00:01  0:00:05 38537
100  240k  100  240k    0     0   192k      0  0:00:01  0:00:01 --:--:--  192k
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxau/libxau_1.0.11_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8488  100  8488    0     0  76945      0 --:--:-- --:--:-- --:--:-- 77163
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.4 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxdmcp/libxdmcp_1.1.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 24020  100 24020    0     0   175k      0 --:--:-- --:--:-- --:--:--  176k
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.15.2-4 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xcb-proto/xcb-proto_1.15.2-4_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 74988  100 74988    0     0  79198      0 --:--:-- --:--:-- --:--:-- 79184
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.15.2-4 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-xcbgen/python-xcbgen_1.15.2-4_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 46516  100 46516    0     0  50289      0 --:--:-- --:--:-- --:--:-- 50287
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.15 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxcb/libxcb_1.15_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 28  893k   28  255k    0     0  1889k      0 --:--:-- --:--:-- --:--:-- 1890k
100  893k  100  893k    0     0  5935k      0 --:--:-- --:--:-- --:--:-- 5916k
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.4.0-9 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xtrans/xtrans_1.4.0-9_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 62 42780   62 26836    0     0  29357      0  0:00:01 --:--:--  0:00:01 29328
100 42780  100 42780    0     0  46711      0 --:--:-- --:--:-- --:--:-- 46703
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.4-1 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libx11/libx11_1.8.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1895k  100 1895k    0     0  11.4M      0 --:--:-- --:--:-- --:--:-- 11.4M
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxrender/libxrender_0.9.11_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 23576  100 23576    0     0  30859      0 --:--:-- --:--:-- --:--:-- 30858
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxft/libxft_2.3.8_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 38 47896   38 18640    0     0  20382      0  0:00:02 --:--:--  0:00:02 20371
100 47896  100 47896    0     0  52258      0 --:--:-- --:--:-- --:--:-- 52231
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.5 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxext/libxext_1.3.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 97068  100 97068    0     0   762k      0 --:--:-- --:--:-- --:--:--  764k
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.3-13 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxss/libxss_1.2.3-13_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 11666  100 11666    0     0   102k      0 --:--:-- --:--:-- --:--:--  101k
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.13 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/tk/tk_8.6.13_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 1701k    0 15901    0     0  17067      0  0:01:42 --:--:--  0:01:42 17061
100 1701k  100 1701k    0     0   980k      0  0:00:01  0:00:01 --:--:--  980k
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.11.3 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python/python_3.11.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 7755k  100 7755k    0     0  39.4M      0 --:--:-- --:--:-- --:--:-- 39.6M
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.11.3 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-tkinter/python-tkinter_3.11.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  7 1189k    7 88794    0     0  80595      0  0:00:15  0:00:01  0:00:14 80575
100 1189k  100 1189k    0     0   754k      0  0:00:01  0:00:01 --:--:--  754k
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.11.3 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-ensurepip-wheels/python-ensurepip-wheels_3.11.3_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2853k  100 2853k    0     0  16.1M      0 --:--:-- --:--:-- --:--:-- 16.2M
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency libxml2@2.11.4 if necessary...
Found libxml2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2/libxml2_2.11.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  515k  100  515k    0     0  3483k      0 --:--:-- --:--:-- --:--:-- 3508k
extracting libxml2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-utils@2.11.4 if necessary...
Found libxml2-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-utils/libxml2-utils_2.11.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 91 30740   91 28208    0     0  37338      0 --:--:-- --:--:-- --:--:-- 37312
100 30740  100 30740    0     0  40664      0 --:--:-- --:--:-- --:--:-- 40661
extracting libxml2-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python@2.11.4 if necessary...
Found libxml2-python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python/libxml2-python_2.11.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  9  198k    9 19996    0     0  21203      0  0:00:09 --:--:--  0:00:09 21204
100  198k  100  198k    0     0   156k      0  0:00:01  0:00:01 --:--:--  156k
extracting libxml2-python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python-static@2.11.4 if necessary...
Found libxml2-python-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python-static/libxml2-python-static_2.11.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 69640  100 69640    0     0  74692      0 --:--:-- --:--:-- --:--:-- 74640
extracting libxml2-python-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland@1.21.0-1 if necessary...
Found libwayland in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland/libwayland_1.21.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  113k  100  113k    0     0   827k      0 --:--:-- --:--:-- --:--:--  823k
extracting libwayland to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-cross-scanner@1.21.0-1 if necessary...
Found libwayland-cross-scanner in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland-cross-scanner/libwayland-cross-scanner_1.21.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 21 80416   21 17263    0     0  19035      0  0:00:04 --:--:--  0:00:04 19033
100 80416  100 80416    0     0  87750      0 --:--:-- --:--:-- --:--:-- 87694
extracting libwayland-cross-scanner to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-protocols@1.31-1 if necessary...
Found libwayland-protocols in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland-protocols/libwayland-protocols_1.31-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 72884  100 72884    0     0   590k      0 --:--:-- --:--:-- --:--:--  593k
extracting libwayland-protocols to /home/builder/.termux-build/_cache-all...
Downloading dependency vulkan-headers@1.3.251 if necessary...
Found vulkan-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-headers/vulkan-headers_1.3.251_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  3 1144k    3 40301    0     0  53203      0  0:00:22 --:--:--  0:00:22 53167
100 1144k  100 1144k    0     0   921k      0  0:00:01  0:00:01 --:--:--  921k
extracting vulkan-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency libxrandr@1.5.3 if necessary...
Found libxrandr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxrandr/libxrandr_1.5.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 21800  100 21800    0     0   188k      0 --:--:-- --:--:-- --:--:--  188k
extracting libxrandr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency vulkan-loader-generic@1.3.251 if necessary...
Found vulkan-loader-generic in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-loader-generic/vulkan-loader-generic_1.3.251_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  102k  100  102k    0     0   797k      0 --:--:-- --:--:-- --:--:--  803k
extracting vulkan-loader-generic to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/KhronosGroup/Vulkan-Tools/archive/v1.3.251.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 53253    0 53253    0     0  76112      0 --:--:-- --:--:-- --:--:-- 76112
100  768k    0  768k    0     0   857k      0 --:--:-- --:--:-- --:--:-- 3655k
Applying patch: no-android.patch
Downloading https://cmake.org/files/v3.26/cmake-3.26.4-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 46.0M    0  128k    0     0   175k      0  0:04:28 --:--:--  0:04:28  175k
 61 46.0M   61 28.4M    0     0  16.4M      0  0:00:02  0:00:01  0:00:01 16.4M
100 46.0M  100 46.0M    0     0  21.3M      0  0:00:02  0:00:02 --:--:-- 21.3M
[termux_setup_cmake]: Applying cmake-c-compiler.patch
Downloading https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  100k  100  100k    0     0   229k      0 --:--:-- --:--:-- --:--:--  229k
Archive:  /home/builder/.termux-build/vulkan-tools/tmp/ninja-1.10.2.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.10.2/ninja  
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The C compiler identification is Clang 14.0.7
-- The CXX compiler identification is Clang 14.0.7
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/aarch64-linux-android-clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using find_package to locate Vulkan
-- Found Vulkan: /data/data/com.termux/files/usr/lib/libvulkan.so  
-- Vulkan FOUND = TRUE
-- Vulkan Lib Dir = /data/data/com.termux/files/usr/lib
-- Vulkan Lib = /data/data/com.termux/files/usr/lib/libvulkan.so
CMake Error at /home/builder/.termux-build/_cache/cmake-3.26.4/share/cmake-3.26/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python3 (missing: Interpreter)

      Reason given by package: 
          Interpreter: Cannot run the interpreter "/data/data/com.termux/files/usr/bin/python3"

Call Stack (most recent call first):
  /home/builder/.termux-build/_cache/cmake-3.26.4/share/cmake-3.26/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /home/builder/.termux-build/_cache/cmake-3.26.4/share/cmake-3.26/Modules/FindPython/Support.cmake:3766 (find_package_handle_standard_args)
  /home/builder/.termux-build/_cache/cmake-3.26.4/share/cmake-3.26/Modules/FindPython3.cmake:551 (include)
  cube/CMakeLists.txt:30 (find_package)


-- Configuring incomplete, errors occurred!
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2023-06-01 00:35:02 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/5138946487">5138946487</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/vulkan-tools/build.sh
+++ b/packages/vulkan-tools/build.sh
@@ -3,11 +3,11 @@ TERMUX_PKG_DESCRIPTION="Vulkan Tools and Utilities"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # This package and vulkan-headers should be updated at same time. Otherwise, they do not compile successfully.
-TERMUX_PKG_VERSION="1.3.250"
+TERMUX_PKG_VERSION="1.3.251"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-Tools/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=a1cd4f7d8d58bec26efb1e5ea73f82dcf36a9662d55401b81a783071b211a35a
-TERMUX_PKG_BUILD_DEPENDS="libwayland-protocols, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-loader-generic (=${TERMUX_PKG_VERSION})"
+TERMUX_PKG_SHA256=03851334d9751464f96b2d3c49cac884b7be5178c04d162a445bb2ca93f56519
 TERMUX_PKG_DEPENDS="libc++, libx11, libxcb, libwayland"
+TERMUX_PKG_BUILD_DEPENDS="libwayland-protocols, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-loader-generic (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_RECOMMENDS="vulkan-loader"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_TAG_TYPE="newest-tag"
@@ -17,6 +17,9 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 -DBUILD_WSI_WAYLAND_SUPPORT=ON
 -DBUILD_WSI_XCB_SUPPORT=ON
 -DBUILD_WSI_XLIB_SUPPORT=ON
+-DPython3_EXECUTABLE=$(command -v python3)
 -DVULKAN_HEADERS_INSTALL_DIR=${TERMUX_PREFIX}
 -DWAYLAND_SCANNER_EXECUTABLE=${TERMUX_PREFIX}/opt/libwayland/cross/bin/wayland-scanner
 "
+
+# https://github.com/KhronosGroup/Vulkan-Tools/blob/main/CMakeLists.txt

